app/src/main/res/values/strings.xml

```diff
@@ -1,6 +1,11 @@
 <resources>
     <string name="app_name">imhungry</string>
 
+    <!-- Notification strings -->
+    <string name="CHANNEL_ID">HUNGRY_CHANNEL</string>
+    <string name="channel_description">imhungry info</string>
+    <string name="channel_name">imhungry notifications</string>
```

Comment:
All of these strings should be more generic, such as `notification_channel_primary_*`. We want to easily be able to add more channels or other components without ambiguity issues.